test/e2e/pods.go

```diff
@@ -30,10 +30,59 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+func runLivenessTest(c *client.Client, podDescr *api.Pod) {
+	defer GinkgoRecover()
+	ns := "e2e-test-" + string(util.NewUUID())
+
+	By(fmt.Sprintf("Creating pod %s in namespace %s", podDescr.Name, ns))
+	_, err := c.Pods(ns).Create(podDescr)
+	Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("creating pod %s", podDescr.Name))
+
+	// At the end of the test, clean up by removing the pod.
+	defer func() {
```

Comment:
Well, or the super awkward to read:

```
defer By("deleting the pod", Expect(
         c.Pods(ns).Delete(podDescr.Name)).NotTo(HaveOccurred()))
```

at that point I'd rather have an English error wrapper for the ErrorNotToHaveOccurred bit so that it looks more like:
`defer By("deleting the pod", RunOrDie(c.Pods(ns).Delete(podDescr.Name)))`